<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PesqIndicacoes</class>
 <widget class="QDialog" name="PesqIndicacoes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtNome"/>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnConfirmar">
     <property name="text">
      <string>&amp;Confirmar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnCancelar">
     <property name="text">
      <string>Cancela&amp;r</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
